### PR TITLE
added h1-tag with ReadMe

### PR DESCRIPTION
**Pull request by asnatschko**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -2,9 +2,10 @@ import Head from 'next/head';
 import Image from 'next/image';
 import styles from '../styles/Home.module.css';
```
Comment (MerleMak) [inline pages/index.js:3]: You can (and should) delete the styles folder and all imports that come with it :)

**Review by MerleMak — COMMENTED** [review]
Looks good so far :)

File: pages/index.js (hunk 1 above)
Comment (asnatschko) [reply pages/index.js]: Done :)

**Review by Murci2022 — COMMENTED** [review]
Hi Alex, it looks great to me.

**Pull request merged**